content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,595 @@
+Cluster/platform-ref-aws (default)                                True     False   Waiting: ...resource claim is waiting for composite resource to become Ready
+```
+
+Use `--output=wide` to see the full message.
+
+```shell {copy-lines="1"
+crossplane trace cluster.aws.platformref.upbound.io platform-ref-aws --output=wide
+NAME                                                              SYNCED   READY   STATUS
+Cluster/platform-ref-aws (default)                                True     False   Waiting: Composite resource claim is waiting for composite resource to become Ready
+```
+
+#### Graphviz dot file output
+
+Use the `--output=dot` to print out a textual 
+[Graphviz dot](https://graphviz.org/docs/layouts/dot/) output. 
+
+Save the output and export it or the output directly to Graphviz `dot` to 
+render an image. 
+
+For example, to save the output as a `graph.png` file use 
+`dot -Tpng -o graph.png`.
+
+`crossplane beta trace cluster.aws.platformref.upbound.io platform-ref-aws -o dot | dot -Tpng -o graph.png`
+
+#### Print connection secrets
+
+Use `-s` to print any connection secret names along with the other resources.
+
+{{<hint "important">}}
+The `crossplane beta trace` command doesn't print secret values.
+{{< /hint >}}
+
+The output includes both the secret name along with the secret's namespace.
+
+```shell
+NAME                                                                        SYNCED   READY   STATUS
+Cluster/platform-ref-aws (default)                                          True     True    Available
+└─ XCluster/platform-ref-aws-mlnwb                                          True     True    Available
+   ├─ XNetwork/platform-ref-aws-mlnwb-6nvkx                                 True     True    Available
+   │  ├─ SecurityGroupRule/platform-ref-aws-mlnwb-szgxp                     True     True    Available
+   │  └─ Secret/3f11c30b-dd94-4f5b-aff7-10fe4318ab1f (upbound-system)       -        -
+   ├─ XEKS/platform-ref-aws-mlnwb-fqjzz                                     True     True    Available
+   │  ├─ OpenIDConnectProvider/platform-ref-aws-mlnwb-h26xx                 True     True    Available
+   │  └─ Secret/9666eccd-929c-4452-8658-c8c881aee137-eks (upbound-system)   -        -
+   ├─ XServices/platform-ref-aws-mlnwb-bgndx                                True     True    Available
+   │  ├─ Release/platform-ref-aws-mlnwb-7hfkv                               True     True    Available
+   │  └─ Secret/d0955929-892d-40c3-b0e0-a8cabda55895 (upbound-system)       -        -
+   └─ Secret/9666eccd-929c-4452-8658-c8c881aee137 (upbound-system)          -        -
+```
+
+### beta xpkg init
+
+The `crossplane beta xpkg init` command populates the current directory with 
+files to build a package. 
+
+Provide a name to use for the package and the package template to start from 
+with the command  
+`crossplane beta xpkg init <name> <template>`
+
+The `<name>` input isn't used. 
```

Comment:
should this be explained further that it isn't currently used, but is reserved for future usage?